### PR TITLE
[ISSUE #3221] record operation logs in admin

### PR DESCRIPTION
**Pull request by li-keguo**
about #3221

Make sure that:

- [x] Optimizing data changes the flow of events.
- [x] Adapt and log data changes.
- [x] Provides a configurable logging API.

Results the following：
image

This PR is not complete, **the initialization script of PG is needed for this submission, I need help.**

Currently, only plugin logs are recorded. In the future, other data changes will be recorded, especially those from bootstrap
I will provide more PR to do that.

### CODE REVIEW

**Comment by KevinClair** [comment]
LGTM.

File: shenyu-admin/src/main/resources/sql-script/mysql/schema.sql [hunk 1]
```diff
@@ -248,6 +248,18 @@ CREATE TABLE IF NOT EXISTS `data_permission` (
     `date_updated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'update time',
     PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC COMMENT='data permission table';
+-- ----------------------------
+-- Table structure for operation_record_log
+-- ----------------------------
+CREATE TABLE IF NOT EXISTS `operation_record_log`
```
Comment (yu199195) [inline shenyu-admin/src/main/resources/sql-script/mysql/schema.sql:254]: add Upgrade log

**Comment by yu199195** [comment]
pls this new feature to dev mail list~

**Comment by li-keguo** [comment]
ok,I deal with it at night. 
I need help with the upgrade script for pg, including tests, can you help me

**Comment by li-keguo** [comment]
Thanks to @ShawnJim for his help with PG

**Review by yu199195 — DISMISSED** [review]

**Review by yu199195 — APPROVED** [review]

**Pull request merged**